guides/common/modules/proc_synchronizing-the-new-repositories.adoc

```diff
@@ -6,41 +6,28 @@ You must enable and synchronize the new {ProjectVersion} repositories before you
-+
-* To upgrade {Project} clients, enable the *{project-client-name}* repositories for all {RHEL} versions that clients use.
-+
-* If you have {SmartProxyServers}, to upgrade them, enable the following repositories too:
-+
-*{ProjectName} {SmartProxy} {ProjectVersion} (for RHEL 9 x86_64) (RPMs)*
-+
-*{ProjectName} Maintenance {ProjectVersion} (for RHEL 9 x86_64) (RPMs)*
-+
-*{EL} 9 (for x86_64 {endash} BaseOS) (RPMs)*
-+
-*{EL} 9 (for x86_64 {endash} AppStream) (RPMs)*
-
-+
-[NOTE]
-====
-If the {ProjectVersion} repositories are not available, refresh the Red{nbsp}Hat Subscription Manifest.
-In the {ProjectWebUI}, navigate to *Content* > *Subscriptions*, click *Manage Manifest*, then click *Refresh*.
-====
-+
+Enable the following repositories:
+* Repositories required to upgrade {SmartProxyServers}:
+** *{ProjectName} {SmartProxy} {ProjectVersion} (for RHEL 9 x86_64) (RPMs)*
+** *{ProjectName} Maintenance {ProjectVersion} (for RHEL 9 x86_64) (RPMs)*
+** *{RHEL} 9 (for x86_64 {endash} BaseOS) (RPMs)*
+** *{RHEL} 9 (for x86_64 {endash} AppStream) (RPMs)*
+* Repositories required to upgrade {Project} clients:
+** *{project-client-name}* for each {RHEL} version your clients are using
 . In the {ProjectWebUI}, navigate to *Content* > *Sync Status*.
-. Click the arrow next to the product to view the available repositories.
-. Select the repositories for {ProjectVersion}.
-Note that *{project-client-name}* does not have a {ProjectVersion} version.
-Choose *{project-client-name}* instead.
-. Click *Synchronize Now*.
+Synchronize the following repositories:
+** All repositories for version {ProjectVersion}
+** *{project-client-name}* repositories for each {RHEL} version your clients are using
+
+.Troubleshooting
+* If you do not see the {ProjectVersion} repositories in the list of repositories or if an error occurs when synchronizing a repository, refresh the Red{nbsp}Hat subscription manifest.
+In the {ProjectWebUI}, navigate to *Content* > *Subscriptions* > *Manage Manifest* > *Refresh*.
 +
-[IMPORTANT]
+[WARNING]
 ====
-If an error occurs when you try to synchronize a repository, refresh the manifest.
-If the problem persists, raise a support request.
-Do not delete the manifest from the Customer Portal or in the {ProjectWebUI}; this removes all the entitlements of your content hosts.
+Do not delete the manifest from the Customer Portal or in the {ProjectWebUI}.
+Deleting your manifest removes all the entitlements of your hosts.
 ====
-+
-. If you use content views to control updates to the base operating system of {SmartProxyServer}, update those content views with new repositories, publish, and promote their updated versions.
-For more information, see {ContentManagementDocURL}Managing_Content_Views_content-management[Managing content views] in _{ContentManagementDocTitle}_.
+
+.Additional resources
+* For more information on synchronizing repositories, see {ContentManagementDocURL}[_{ContentManagementDocTitle}_].
```

Comment:
The guide is huge. Can we use a more specific link to the _Importing content_ chapter?